query/src/sql/planner/binder/sort.rs

```diff
@@ -148,14 +148,28 @@ impl<'a> Binder {
         let mut order_by_items = Vec::with_capacity(order_by.items.len());
         let mut scalars = vec![];
         for order in order_by.items {
+            if from_context.in_grouping {
```

Comment:
Maybe we need do some modify to fix this case 

```sql
select distinct(id) as id1 from t order by id1;
```

Because the in_grouping default value if false , only when the sql has aggregate it will be set true.

Reply:
I think this should create a new issue? what do you think about it?@[leiysky](https://github.com/leiysky)

Reply:
@TCeason I think so. 

I have no idea about this case yet, since we haven't support `DISTINCT ON` yet, this query should be the same as `select distinct id as id1 from t order by id1`.

Reply:
Np  :) I think that’s not a regular usage. We just need record it now.